cypress/e2e/specialFeatures/atiAnalytics/helpers/index.js

```diff
@@ -92,6 +93,45 @@ export const interceptATIAnalyticsBeacons = () => {
     ).as(`${component}-ati-click`);
   });
 
+  // Component Views & Clicks - Viewability Model
+  Object.values(COMPONENTS).forEach(component => {
+    const viewEventRegex = new RegExp(
+      `\\[\\{"name":"viewability\\.view","data":\\{(?:.*)?"event":\\{"category":"viewability","action":"view"\\}(?:.*)?"item":\\{(?:.*)?"name":"${component}(.*)?"(?:.*)?\\}\\}\\}\\]`,
+      'g',
+    );
+
+    const clickEventRegex = new RegExp(
+      `\\[\\{"name":"viewability\\.select","data":\\{(?:.*)?"event":\\{"category":"viewability","action":"select"\\}(?:.*)?"item":\\{(?:.*)?"name":"${component}(.*)?"(?:.*)?\\}\\}\\}\\]`,
+      'g',
+    );
+
+    // Component Views
+    cy.intercept(
+      {
+        url: `${atiUrl}/*`,
+        query: {
+          events: viewEventRegex,
+        },
+      },
+      request => {
+        request.reply({ statusCode: 200 });
+      },
+    ).as(`${component}-ati-view-viewability`);
+
+    // Component Clicks
+    cy.intercept(
+      {
+        url: `${atiUrl}/*`,
+        query: {
+          events: clickEventRegex,
+        },
+      },
+      request => {
+        request.reply({ statusCode: 200 });
+      },
+    ).as(`${component}-ati-click-viewability`);
+  });
```

Comment:
I've duplicated the [CPV model logic](https://github.com/bbc/simorgh/pull/12648/files#diff-fa84316e05037b975ddeaf22c65567eb5c25137e6f8c8b1d2a9d9719f38d61f8R62-R94) instead of abstracting the matchers
into a function to retain the verbose nature of the tests.